core/ir/x86/opcode_api.h

```diff
@@ -1692,4 +1692,8 @@ enum {
 #define OP_icebp OP_int1
 #define OP_setalc OP_salc
 
+/* renamed opcodes */
```

Comment:
style nit: Prefer capitalized + punctuated for new comments esp whole-line comments.